src/app/login-page/login-page.component.ts

```diff
@@ -34,12 +34,12 @@ export class LoginPageComponent implements OnInit {
 
   
   login(data: any) {
-    this.serv.login(data).subscribe(d => {
+    this.serv.login(data).subscribe() 
```

Comment:
this is wrong, how come you'll move the below 3 lines of code outside subscribe, there is a reason why we wrote those 3 lines inside subscribe, explain to me tomorrow why we have written like that

Reply:
resolved